### PR TITLE
Bump AGP versions

### PR DESCRIPTION
**Pull request by romtsn**

### CODE REVIEW

**Comment by romtsn** [comment]
blocked by #2563

File: buildSrc/src/main/java/Config.kt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import java.math.BigDecimal
 
 object Config {
-    val AGP = System.getenv("VERSION_AGP") ?: "7.3.0"
+    val AGP = System.getenv("VERSION_AGP") ?: "7.4.1"
```
Comment (romtsn) [inline buildSrc/src/main/java/Config.kt:4]:
```suggestion
    val AGP = System.getenv("VERSION_AGP") ?: "7.4.2"
```

File: .github/workflows/agp-matrix.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        agp: ['7.3.0','7.4.0-rc01','8.0.0-alpha09']
+        agp: ['7.4.1','8.0.0-beta03','8.1.0-alpha06']
```
Comment (romtsn) [inline .github/workflows/agp-matrix.yml:24]:
```suggestion
        agp: ['8.0.0-rc01','8.1.0-alpha11']
```

**Review by markushi — APPROVED** [review]
Nice!

**Pull request merged**